rasa/utils/train_utils.py

```diff
@@ -95,22 +95,24 @@ def align_tokens(
     current_token_offset = token_start
 
     for index, string in enumerate(tokens_in):
+        string_len = len(string) if string != unk_token else 1
```

Comment:
```suggestion
        string_len = len(string) if unk_token is None or string != unk_token else 1
```

Reply:
Also can you please add a comment why `1` to the code? Thanks.

Reply:
Good suggestion! I already have done that in a new commit!